suite-common/wallet-utils/src/transactionUtils.ts

```diff
@@ -923,7 +923,11 @@ export const simpleSearchTransactions = (
     // Find by token name, symbol or contract
     const foundTxsForToken = transactions.flatMap(transaction => {
         const hasMatchingToken = transaction.tokens.some(token => {
-            return isTokenMatchesSearch(token, search.toLowerCase());
+            return (
+                isTokenMatchesSearch(token, search.toLowerCase()) ||
+                token.to?.toLowerCase().includes(search) ||
+                token.from?.toLowerCase().includes(search)
```

Comment:
it is not working because you are missing `search.toLowerCase()`

Reply:
Sorry, I was thinking about doing it here:

`search = search.trim();`

but now sure if it can break something else with date etc. search